bin/cli

```diff
@@ -51,18 +53,23 @@ async function updatePackageJson (packageJsonPath) {
 }
 
 async function initialiseGitRepo () {
-  const presenceCheckOutput = []
-  const notPresentText = 'not present'
-  await exec(`which git || echo "${notPresentText}"`, {}, data => presenceCheckOutput.push(data))
-  if (presenceCheckOutput.join('').trim() === notPresentText) {
+  if (argv.options['no-version-control']) {
+    return
+  }
+  try {
+    await exec('git init && git checkout -b main && git add -A .', {})
```

Comment:
This might be better with `git branch -m main`? That way if there is a branch named `master` it won't persist.

```suggestion
    await exec('git init && git branch -m main && git add -A .', {})
```

Reply:
I've used an option on init to avoid needing to fix it afterwards. `git init --initial-branch=main`